### PR TITLE
Update hackclub.com.yaml

### PR DESCRIPTION
**Pull request by tensixxx**
# [Adding/Deleting/Updating] `subdomain.[hackclub.com/dino.icu/etc]`

## Description

Provide a description here of the changes, and if adding a subdomain provide a rationale & what it will be used for.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2018,6 +2019,10 @@ pic._domainkey.mail:
   ttl: 600
   type: TXT
   value: k=rsa
+  pinetree.hack.com:
+  - ttl: 600
+    type: A
+    value: 127.0.0.1.
```
Comment (Muirrum) [inline hackclub.com.yaml:2025]: couple things here:


```suggestion
pinetree:
  - ttl: 600
    type: A
    value: 127.0.0.1.
```

127.0.0.1 is the loopback address, what do you want this domain to point to?

**Review by Muirrum — CHANGES_REQUESTED** [review]
In addition, can you give details on the purpose of this domain?

**Comment by tensixxx** [comment]
Dear Sir,

I will use this subdomain (pinetree.hack.com) as a school hack club's website.
I don't really know what I should do to replace the IP address.
Could you tell me what I should do?

Thanks,
Rachel
________________________________
发件人: Cara Salter ***@***.***>
发送时间: 2024年7月12日 18:41
收件人: hackclub/dns ***@***.***>
抄送: tensixxx ***@***.***>; Author ***@***.***>
主题: Re: [hackclub/dns] Update hackclub.com.yaml (PR #1221)


@Muirrum requested changes on this pull request.

In addition, can you give details on the purpose of this domain?

________________________________

In hackclub.com.yaml<https://github.com/hackclub/dns/pull/1221#discussion_r1675666497>:

> +  pinetree.hack.com:
+  - ttl: 600
+    type: A
+    value: 127.0.0.1.


couple things here:

⬇️ Suggested change

-  pinetree.hack.com:
-  - ttl: 600
-    type: A
-    value: 127.0.0.1.
+pinetree:
+  - ttl: 600
+    type: A
+    value: 127.0.0.1.


127.0.0.1 is the loopback address, what do you want this domain to point to?

**Comment by Muirrum** [comment]
Hey,

On 7/12/24 5:45 PM, tensixxx wrote:
> Dear Sir,

Not a "sir".
> 
> I will use this subdomain (pinetree.hack.com) as a school hack club's 
> website.
> I don't really know what I should do to replace the IP address.
> Could you tell me what I should do?

That depends on where your website is being hosted. Is it being hosted 
on Vercel, or GitHub pages, or something else? Where does your site live?

Cheers,
Cara

**Comment by Muirrum** [comment]
Following up on this

**Pull request closed without merging**